docker-local/docker-compose.yml

```diff
@@ -350,10 +350,10 @@ services:
       - "dfspa-backend:172.17.0.1"
 
   dfspa-thirdparty-scheme-adapter-inbound:
-    image: "mojaloop/thirdparty-scheme-adapter:v11.25.2"
-    # build:
-    #   context: ../../thirdparty-scheme-adapter
-    #   dockerfile: ./docker/Dockerfile
+    # image: "mojaloop/thirdparty-scheme-adapter:v11.21.0"
+    build:
+      context: https://github.com/kleyow/thirdparty-scheme-adapter.git#feat/pisp-outbound-handler
```

Comment:
Oh. No this is just me getting everything ready so I can bang out merges once the changes are checked in the dependencies. Not sure we want to be building from branches :thinking: but it might be useful in a workflow.